packages/unity-interface/BrowserInterface.ts

```diff
@@ -564,8 +587,8 @@ export class BrowserInterface {
     getUnseenMessagesByUser()
   }
 
-  public SetHomeScene(data: { sceneId: string }) {
-    store.dispatch(setHomeScene(data.sceneId))
+  public SetHomeScene(data: { sceneCoords: string }) {
+    store.dispatch(setHomeScene(data.sceneCoords))
```

Comment:
Will this continue to work until the unity-renderer counterpart is merged?

Reply:
Nice catch, actually in production that is mistakenly called `sceneId` even though the renderer is [sending just the scene coords as a string](https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HomePoint/WebInterfaceHomeLocationController.cs#L8). And in the [counterpart renderer PR](https://github.com/decentraland/unity-renderer/pull/2977) I corrected that.

I just updated this Kernel PR to allow the data with both names, and after merging the renderer  counterpart PR we can then remove the misleading `sceneId` one here.

I tested the "set home" functionality working with both prod renderer and the counterpart PR renderer.